src/main/java/io/prometheus/cloudwatch/CloudWatchCollector.java

```diff
@@ -309,6 +309,15 @@ private void scrape(List<MetricFamilySamples> mfs) {
 
         String unit = null;
 
+        if (rule.awsNamespace.equals("AWS/DynamoDB") && rule.awsDimensions.contains("GlobalSecondaryIndexName")) {
```

Comment:
The metrics that can currently result in duplicate help text are the following:

 * ConsumedReadCapacityUnits
 * ConsumedWriteCapacityUnits
 * ProvisionedReadCapacityUnits
 * ProvisionedWriteCapacityUnits
 * ReadThrottleEvents
 * WriteThrottleEvents

There are also some metrics that should only ever appear in [TableName, GlobalSecondaryIndexName], never in just [TableName].

 * OnlineIndexConsumedWriteCapacity
 * OnlineIndexPercentageProgress
 * OnlineIndexThrottleEvents

My thinking was that it was desirable to have the suffix on all index-related metrics. Would it be better to attach it only for the ones that are potentially duplicates?